mlflow/environment_variables.py

```diff
@@ -0,0 +1,12 @@
+"""
+This module defines environment variable names used in MLflow.
+"""
+
+#: Specify maximum retries for mlflow rest API http request.
+MLFLOW_HTTP_REQUEST_MAX_RETRIES = "MLFLOW_HTTP_REQUEST_MAX_RETRIES"
```

Comment:
```suggestion
MLFLOW_HTTP_REQUEST_MAX_RETRIES = EnvironmentVariable("MLFLOW_HTTP_REQUEST_MAX_RETRIES", int, 5)
```

Can we use `EnvironmentVariable` here?

Reply:
MLflow users (including us) can use this object like this:

```
from mlflow.environment_variables import MLFLOW_HTTP_REQUEST_MAX_RETRIES

print(MLFLOW_HTTP_REQUEST_MAX_RETRIES.get())
```

Reply:
good idea!